internal/tests/TopicBrowsePermissionsResource/topic_browse_permissions_resource_test.go

```diff
@@ -0,0 +1,37 @@
+package TopicBrowsePermissionsResource
+
+import (
+	. "axual.com/terraform-provider-axual/internal/tests"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestTopicBrowsePermissionsResource(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: GetProviderConfig(t).ProtoV6ProviderFactories,
+		ExternalProviders:        GetProviderConfig(t).ExternalProviders,
+
+		Steps: []resource.TestStep{
+			{
+				Config: GetProvider() + GetFile("axual_topic_browse_permissions_initial.tf"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrPair("axual_topic_browse_permissions.tf-test-topic-browse-permissions", "users.0", "axual_user.ben", "id"),
```

Comment:
Maybe we can define a constant for the resourceName to avoid repetition.

Reply:
I understand why this might be a decent idea. However, I really don't like adding repetitive code that increases cognitive complexity in 41 places.

In 41 places we need to add code like this:
```
const (
	applicationResourceName   = "axual_application"
	applicationResourceID     = "tf-test-app"
)
...
resource.TestCheckResourceAttr(applicationResourceName+"."+applicationResourceID, "name", resourceID),
```

Sorry, but I don't like this look of this: yes we slightly protect ourselves from typos but we introduce lots of unnecessary code and string concatenation.